src/pages/Upload.js

```diff
@@ -314,11 +313,8 @@ const WorkspaceSelectorPanel = ({
   ])
 }
 
-const CollectionSelectorPanel = _.flow(
-  withDisplayName('CollectionSelectorPanel'),
-  requesterPaysWrapper({ onDismiss: ({ onClose }) => onClose() })
```

Comment:
The `onClose` prop used by `requesterPaysWrapper` in these panels was never wired up. You shouldn't encounter requester pays errors here though because you can only upload to workspaces you have write access to.